core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -357,34 +357,165 @@ public void wipeInMemorySettings() {
     }
   }
 
-  public <T> void configure(@Nonnull final MongoNamespace namespace,
-                            @Nullable final ConflictHandler<T> conflictHandler,
-                            @Nullable final ChangeEventListener<T> changeEventListener,
-                            @Nullable final ExceptionListener exceptionListener,
-                            @Nonnull final Codec<T> codec) {
+  public boolean hasNamespaceListener(@Nonnull final MongoNamespace namespace) {
     this.waitUntilInitialized();
+    return instanceChangeStreamListener.hasNamespace(namespace);
+  }
+
```

Comment:
Added these methods for testing purposes

Reply:
[nit] Please add a docString here explaining this. I'm not really keen on this style of testing, but it's fine for now.